### PR TITLE
DDF-3839 Added WFS 1.1.0 Source ITests

### PR DESCRIPTION
**Pull request by AzGoalie**
#### What does this PR do?
Adds WFS 1.1.0 Source ITests
#### Who is reviewing it? 
@mojogitoverhere @gordocanchola @GabrielFabian @dcruver 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged)
#### Select relevant component teams: 
https://github.com/orgs/codice/teams/test
#### Ask 2 committers to review/merge the PR and tag them here. If you don't know who to ask, you can request reviews in https://groups.google.com/forum/#!forum/ddf-developers .
(please choose ONLY two committers from below, delete the rest)
@jlcsmith 
#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/test/itests/test-itests-common/src/main/resources/TestSpatial/xml/WFS_11_sfRoads.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- /**
+ * Copyright (c) Codice Foundation
```
Comment (jlcsmith) [inline distribution/test/itests/test-itests-common/src/main/resources/TestSpatial/xml/WFS_11_sfRoads.xml]: ✏️ technically we shouldn't stick codice copyright on data that isn't our own (in this case we just grabbed it from geoserver).  We could add an exception so checkstyle skips these files.

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestSpatial.java [hunk 2]
```diff
@@ -237,6 +262,12 @@ private static String removeFileExtension(String file) {
   public void beforeExam() throws Exception {
     try {
       waitForSystemReady();
+
+      setupMockServer();
+
+      getCatalogBundle().waitForFederatedSource(WFS_11_SOURCE_ID);
+      //      getServiceManager().waitForSourcesToBeAvailable();
```
Comment (jlcsmith) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestSpatial.java]: ✏️ remove comment if not needed

**Comment by jlcsmith** [comment]
build now

File: distribution/test/itests/test-itests-common/src/main/resources/TestSpatial/xml/WFS_11_sfRoads.xml (hunk 1 above)
Comment (AzGoalie) [reply distribution/test/itests/test-itests-common/src/main/resources/TestSpatial/xml/WFS_11_sfRoads.xml]: Done

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestSpatial.java (hunk 2 above)
Comment (AzGoalie) [reply distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestSpatial.java]: Done

**Comment by jlcsmith** [comment]
build now

**Pull request merged**